modules/adding-new-extension-dynamic-plug-in.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * web_console/dynamic-plug-ins.adoc
+
+:_content-type: PROCEDURE
+[id="adding-new-extension-dynamic-plugin_{context}"]
+= Adding a new extension to your plug-in
+You can add different extensions to your plug-in that are loaded at runtime.
```

Comment:
Do you have any suggestions to add to this blurb?

Reply:
> Do you have any suggestions to add to this blurb?

I think it would be best to link later to the list of extension points (when available). For now, we can say something like here is how you can add an example extension to your plugin.